paper-api/src/main/java/org/bukkit/EntityEffect.java

```diff
@@ -252,177 +273,227 @@ public enum EntityEffect {
     RAVAGER_STUNNED(39, Ravager.class),
     /**
      * Cat taming failed.
+     *
+     * @deprecated use {@link #TRUSTING_FAILED}
      */
+    @Deprecated(since = "1.14", forRemoval = true)
     CAT_TAME_FAIL(40, Cat.class),
     /**
      * Cat taming succeeded.
+     *
+     * @deprecated use {@link #TRUSTING_SUCCEEDED}
      */
+    @Deprecated(since = "1.14", forRemoval = true)
     CAT_TAME_SUCCESS(41, Cat.class),
+    /**
+     * Ocelot trusting failed.
+     */
+    TRUSTING_FAILED(40, Ocelot.class),
+    /**
+     * Ocelot trusting succeeded.
+     */
+    TRUSTING_SUCCEEDED(41, Ocelot.class),
     /**
      * Villager splashes particles during a raid.
      */
     VILLAGER_SPLASH(42, Villager.class),
     /**
      * Player's bad omen effect removed to start or increase raid difficult.
+     *
      * @deprecated raid system was overhauled in 1.20.5
      */
     @Deprecated(since = "1.20.5", forRemoval = true)
     PLAYER_BAD_OMEN_RAID(43, Player.class),
     /**
      * Entity hurt due to berry bush. Prickly!
+     *
      * @deprecated in favor of {@link LivingEntity#playHurtAnimation(float)} or {@link Entity#broadcastHurtAnimation(java.util.Collection)}
      */
     @Deprecated(since = "1.19.4", forRemoval = true)
     HURT_BERRY_BUSH(44, LivingEntity.class),
     /**
-     * Fox chews the food in its mouth
+     * Fox chews the food in its mouth.
      */
     FOX_CHEW(45, Fox.class),
     /**
-     * Entity teleported as a result of chorus fruit or as an enderman
+     * Entity teleported as a result of chorus fruit or as an enderman.
      */
     TELEPORT_ENDER(46, LivingEntity.class),
     /**
-     * Entity breaks item in main hand
+     * Entity breaks item in main hand.
```

Comment:
Why do the other eqipments get a see to their equipment slot but these two don't?